staging/src/k8s.io/apiserver/pkg/util/proxy/proxy.go

```diff
@@ -17,17 +17,29 @@ limitations under the License.
 package proxy
 
 import (
+	"context"
 	"fmt"
 	"math/rand"
 	"net"
+	"net/http"
 	"net/url"
 	"strconv"
+	"strings"
+	"time"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/errors"
+	utilnet "k8s.io/apimachinery/pkg/util/net"
+	auditinternal "k8s.io/apiserver/pkg/apis/audit"
+	"k8s.io/apiserver/pkg/audit"
+	genericapirequest "k8s.io/apiserver/pkg/endpoints/request"
 	listersv1 "k8s.io/client-go/listers/core/v1"
 )
 
+const (
+	aggregatedDiscoveryTimeout = 5 * time.Second
```

Comment:
How was the selected?

Reply:
taken from https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kube-aggregator/pkg/apiserver/handler_proxy.go#L47

Reply:
This is the only single constant that's being used everywhere.

Reply:
add a comment explaining that you are taking this variable from  https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kube-aggregator/pkg/apiserver/handler_proxy.go#L47, all these decisions should be reflected in the code as nobody will be able to read this comments once it merges

Reply:
Added